### PR TITLE
[Tokenizers] Try to fix macOS build

### PR DESCRIPTION
**Pull request by ilya-lavrenov**

### CODE REVIEW

File: .ci/azure/mac.yml [hunk 1]
```diff
@@ -105,53 +106,53 @@ jobs:
...
         -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) \
         -DBUILD_nvidia_plugin=OFF \
         -DOPENVINO_EXTRA_MODULES=$(REPO_DIR)/modules \
-        -DCMAKE_OSX_DEPLOYMENT_TARGET=11.0 \
         -DCMAKE_OSX_ARCHITECTURES=$(CMAKE_OSX_ARCHITECTURES) \
         -DENABLE_OV_PADDLE_FRONTEND=OFF \
         -DENABLE_OV_TF_FRONTEND=OFF \
         -DENABLE_OV_TF_LITE_FRONTEND=OFF \
         -DENABLE_OV_PYTORCH_FRONTEND=OFF \
         -DENABLE_CPPLINT=OFF \
```
Comment (ilya-lavrenov) [inline .ci/azure/mac.yml:135]:
```suggestion
        -DENABLE_CPPLINT=OFF \
        -DPython3_EXECUTABLE=`which python3.10` \
```

**Comment by apaniukov** [comment]
Seems like master is working fine: https://dev.azure.com/openvinoci/dldt/_build/results?buildId=679488&view=logs&j=6c23757f-dbf6-510b-3bd6-0534daaa0edf
Should we close this PR?

**Pull request closed without merging**